docs/UserGuide.adoc

```diff
@@ -821,8 +821,8 @@ e.g. `findpolicy senior`
 e.g. `findtagpeople smoker diabetic`
 * *Find policy by tags*: `findtagpolicy TAG [MORE_TAGS]` +
 e.g. `findtagpolicy accident life`
-* *Assign Policy* : `assignpolicy perindex/INDEX polindex/INDEX` +
-e.g. `assignpolicy perindex/2 polindex/2`
+* *Assign Policy* : `assignpolicy INDEX pol/POLICY NAME` +
+e.g. `assignpolicy 2 pol/health`
```

Comment:
Small thing but maybe we can change 'health' to 'Health Insurance' to be clear we don't assign partial matches?